ibm/mas_devops/roles/mongodb/tasks/providers/community/install.yml

```diff
@@ -1,6 +1,51 @@
 ---
 
-# 1. Debug Properties
+# 1. Check for undefined properties that do not have a default
+# -----------------------------------------------------------------------------
+- name: "Fail if catalog_tag is not provided"
+  assert:
+    that: catalog_tag is defined and catalog_tag != ""
```

Comment:
Can we call it mas_catalog_version as that is what it is rather than introducing a different name